Classes/Hooks/ImageTextLinkPreviewRenderer.php

```diff
@@ -0,0 +1,47 @@
+<?php
+namespace T3kit\themeT3kit\Hooks;
+
+class ImageTextLinkPreviewRenderer implements \TYPO3\CMS\Backend\View\PageLayoutViewDrawItemHookInterface
+{
+	/**
+	 * Preprocesses the preview rendering of the custom content element which has image and media image
+	 *
+	 * @param \TYPO3\CMS\Backend\View\PageLayoutView $parentObject Calling parent object
+	 * @param bool $drawItem Whether to draw the item using the default functionalities
+	 * @param string $headerContent Header content
+	 * @param string $itemContent Item content
+	 * @param array $row Record row of tt_content
+	 * @return void
+	 */
+	public function preProcess(\TYPO3\CMS\Backend\View\PageLayoutView &$parentObject, &$drawItem, &$headerContent, &$itemContent, array &$row) {
+		if ($row['CType'] === 'imageTextLink') {
+			if ($row['bodytext']) {
+				$itemContent .= $parentObject->linkEditContent($parentObject->renderText($row['bodytext']), $row) . '<br />';
+			}
+
+			if ($row['image']) {
+				$itemContent .= $parentObject->linkEditContent($parentObject->getThumbCodeUnlinked($row, 'tt_content', 'image'), $row);
+				$fileReferences = \TYPO3\CMS\Backend\Utility\BackendUtility::resolveFileReferences('tt_content', 'image', $row);
```

Comment:
@chantheoun I think there is no specific rule defined. According to [contribution guidelines](https://github.com/t3kit/t3kit/blob/master/CONTRIBUTING.md) we should follow TYPO3 coding guidelines which say that use statement may be used.

PHP developers defacto used by code sniffers is that we should use `use` instead of full path.

@MattiasNilsson and @jozefspisiak is there preference among t3kit developers on the issue?